test/artificial_samples/makefile

```diff
@@ -36,7 +36,7 @@ endef
 
```

Comment:
can't we change everything to dockcross?

Reply:
Done.